executor/admin_test.go

```diff
@@ -1095,7 +1095,7 @@ func TestCheckFailReport(t *testing.T) {
 		require.NoError(t, txn.Commit(tk.ctx))
 
 		ctx, hook := withLogHook(tk.ctx, t, "inconsistency")
-		_, err = tk.Exec(ctx, "admin check table admin_test")
+		res, err := tk.Exec(ctx, "admin check table admin_test")
```

Comment:
How about use `MustGetErrCode`?

Reply:
It is not an testkit so that it cannot use ```MustGetErrMsg```.

it just looks like ```testkit```.

Reply:
BTW. closing this resultset causes the test to fail. so I have removed this.